lib/syscall_shim/include/uk/legacy_syscall.h

```diff
@@ -0,0 +1,27 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
Shouldn't this be `BSD-3-Clause`?

Reply:
Actually I am not an expert into this, I see that we have BSD-2 on some files (e.g. https://github.com/unikraft/unikraft/blob/staging/arch/x86/x86_64/include/uk/asm/limits.h)  and BSD-3 on other files  (e.g. https://github.com/unikraft/unikraft/blob/staging/lib/syscall_shim/include/uk/syscall.h)  . For now I will switch to BSD-3 here for consistency in the same directory (and also add the missing copyright in the header).